packages/bits/demo/src/components/demo/wizard-v2/wizard-with-custom-icons/wizard-with-custom-icons-example.component.ts

```diff
@@ -1,6 +1,32 @@
 import { Component } from "@angular/core";
+import { IWizardConfig, IWizardStepStateIconConfig, WIZARD_CONFIG } from "@nova-ui/bits";
 @Component({
     selector: "nui-wizard-with-custom-icons-example",
     templateUrl: "./wizard-with-custom-icons-example.component.html",
+    providers: [
+        {
+            provide: WIZARD_CONFIG,
+            useValue: {
+                stepStateIcons: {
```

Comment:
Maybe `stepStateIconsConfig` for better clarity that this is a configuration object rather than a collection of icons?

Reply:
It seems logical to me that what's inside the config is a config. I'm not sure we should explicitly add `config` to config keys if they're part of the... config :)

Reply:
okay